build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+python src/etc/build_rust.py $@
```

Comment:
@nagisa  @huonw  I ran the `2to3` tool and added `from __future__ import print_function`. I tested against both python 2.7 and 3.4 and it works on both versions.